examples/common/imgui_ui/windows/boolean_state.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#include "boolean_state.h"
+
+#include <imgui.h>
+
+#include <math.h>
+
+#include <app-common/zap-generated/attributes/Accessors.h>
+#include <app-common/zap-generated/cluster-enums.h>
+
+namespace example {
+namespace Ui {
+namespace Windows {
+
+void BooleanState::UpdateState()
+{
+    if (mTargetState.HasValue())
+    {
+        chip::app::Clusters::BooleanState::Attributes::StateValue::Set(mEndpointId, mTargetState.Value());
```

Comment:
Should this be sending the StateChange event, at least if the app claims to support it?  This is why we should ideally have a cluster API instead of poking attributes directly....

Reply:
Added a TODO for now - we do not have clusters/boolean-state to have a cluster API yet.